yarn-project/sequencer-client/package.json

```diff
@@ -30,6 +30,7 @@
     "@aztec/foundation": "workspace:^",
     "@aztec/l1-artifacts": "workspace:^",
     "@aztec/merkle-tree": "workspace:^",
+    "@aztec/noir-contracts.js": "workspace:^",
```

Comment:
What does everyone thing of this change? (sequencer depending on contracts)

It allows me to add sane defaults in the config and to use the new `Contract.storage.<thing>.slot` for the gas token balance